examples/ses/template.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: ses.aws.crossplane.io/v1alpha1
+kind: Template
+metadata:
+    name: example
+spec:
+    forProvider:
+    region: us-east-2
```

Comment:
indent this section under `forProvider`, and check that it works

Reply:
Indented the section under `forProvider` . 
Checked for Create, Update and Delete operations - Works successfully.